src/ui/components/UploadScreen/UploadScreen.tsx

```diff
@@ -30,9 +30,13 @@ function DeletedScreen({ url }: { url: string }) {
     window.location.href = url;
   };
   useEffect(() => {
-    setTimeout(() => {
+    const timeOutID = setTimeout(() => {
       navigateToUrl();
     }, 1200);
+
+    return () => {
+      clearTimeout(timeOutID);
+    };
```

Comment:
This component looks like it was kind of broken before since this effect had no dependencies. The added change looks okay but I think we also should fix the effect so that it doesn't re-run the effect every time it happens to be rendered. (This is for _us_ to follow up on.)